### ISSUE TITLE
bug: gcds-signature changes style ignoring variant prop creating accessibility contrast issues

### ISSUE THREAD

**Issue by s-laugh** [cds-snc/gcds-components#1113]
### Prerequisites | Prérequis

- [x] I have read the [Contributing Guidelines](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md). | J’ai lu les [Lignes directrices sur les contributions](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md).
- [x] I agree to follow the [Code of Conduct](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md). | J’accepte de me conformer au [Code de conduite](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md).
- [x] I have searched for [existing issues](https://github.com/cds-snc/gcds-components/issues) that already report this problem, without success. | J’ai vérifié parmi [les tickets (issues) existants](https://github.com/cds-snc/gcds-components/issues) que ce sujet n’avait pas déjà été soumis.

### GC Design System Components Package and Version | Paquet et version des composants de Système de design GC

gcds-components@latest (v0.47.0)

### Current Behavior | Comportement observé

The color of the logs is switching between white and black depending on the computers theme settings, even though the variant is set to color.

Windows dark theme:

<img width="480" height="453" alt="Image" src="https://github.com/user-attachments/assets/d4ee2e8e-ee2b-4dae-a85d-1fa173ee2e27" />

Windows light theme:

<img width="483" height="456" alt="Image" src="https://github.com/user-attachments/assets/b24be457-3ba0-4840-9d00-8b69f4fdb2d5" />

### Expected Behavior | Comportement attendu

The variant should accept a "windows theme" option if this functionality is intended, otherwise it should respect the variant selected. - Should never assume the default is to accept another inheriting theme.

_Should ensure all components and background adjust to the theme before embracing this._

### System Info | Information sur le système

```shell
Found in Edge and Chrome on Windows 11.
```

### Steps to Reproduce | Étapes pour reproduire le bogue

Load the component on a windows computer and toggle the windows theme.
